src/App.js

```diff
@@ -21,6 +24,13 @@ function App() {
 
 
 const [todoList,setTodoList] = useSemiPersistentState();
+
+const removeTodo = (id)=>{
+   const newList = todoList.filter(item =>
```

Comment:
This could be reduced to: 
```suggestion
     function removeTodo(id) {
         setTodoList(todoList.filter((todo) => todo.id !== id));
     }

```

Both are right.